### DISCUSSION TITLE
Setup Questions

### DISCUSSION THREAD

**Discussion by KevinKTM** [crocodilestick/Calibre-Web-Automated#97] [General]
1. Is this intended to be used concurrently with a regular calibre docker (assuming only either instance is running but not both at the same time)? If not I assume setup the library with regular calibre then move CWA?
2. Have a bunch of books with calibre generated covers. Can pull the metadata but the cover doesn't upgrade. What am I doing wrong?
3. I have a bunch of duplicated books - how can I delete the duplicates? How does CWA handle duplicates if I put the same book into my incoming folder multiple times, does it get imported as another copy (have a bit of issue as I get books from myanonymouse (torrent} and they require minimum seed time plus encourage seeding to build points so it's difficult to manage merging a bunch of books that remain seeding and get them into calibre. It's gross.)

**Comment by jalanmac** [comment]
Expanding on #1 above - my current configuration is a local calibre instance with an automated sync to the NAS where a docker instance of Calibre-Web operates on that copy of the calibre database. If I understand CWA setup, it will attempt to automagically convert non-epub formats to epub - so it doesn't appear to be a good candidate to replace Calibre-Web where I would be replacing the underlying database frequently based on my local calibre updates. It seems as if CWA is best used to replace both local calibre and Calibre-Web in my configuration. Is my understanding somewhat accurate? Thanks in advance!
